### PR TITLE
Psp 2371

### PR DESCRIPTION
**Pull request by devinleighsmith**
add basic navigation for creating contacts.

### CODE REVIEW

**Review by jdtoombs — APPROVED** [review]

**Review by FuriousLlama — APPROVED** [review]

File: etl/LISOS_PIMS/LISOS_PIMS/LISOS_PIMS.dtproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <DeploymentModel>Project</DeploymentModel>
-  <ProductVersion>15.0.2000.170</ProductVersion>
+  <ProductVersion>15.0.2000.166</ProductVersion>
```
Comment (FuriousLlama) [inline etl/LISOS_PIMS/LISOS_PIMS/LISOS_PIMS.dtproj:4]: these changes don't seem related to the task. Do they need to be included?

Comment (devinleighsmith) [reply etl/LISOS_PIMS/LISOS_PIMS/LISOS_PIMS.dtproj:4]: no, will remove.
